Semana-18/FutureBook/src/data/postDatabase.ts

```diff
@@ -0,0 +1,34 @@
+import { PostGateway } from './../business/gateways/Post/post';
+import { DatabaseBoilerplate } from "./databaseBoilerplate";
+import { Post } from '../business/entities/post';
+
+
+export class PostDatabase extends DatabaseBoilerplate implements PostGateway {
+
+  private static TABLE_NAME: string = "posts_futurebook"
+
+  public async createNewPost(input: Post): Promise<void> {
+    try {
```

Comment:
A query do SQL ficou bem fácil de entender, parabéns! 
Só quero dizer que nada adianta um `catch` que só joga o erro. Se for usar um bloco `try-catch`, faça um tratamento mínimo do erro!